### ISSUE TITLE
Ability to finish full body calibration with just one controller

### ISSUE THREAD

**Issue by Geenz** [Neos-Metaverse/NeosPublic#3580]
### Is your tweak request related to a problem? Please describe.

Users with disabilities or injuries that prevent them from using one of their arms presently are unable to utilize full body tracking due to how the full body calibrator works.

### Describe what would you like tweaked

The ability to complete full body calibration with a single controller, ether with an accessibility mode or by default.

### Describe alternatives you've considered

None
